beginner_source/blitz/cifar10_tutorial.py

```diff
@@ -32,8 +31,7 @@
 ‘dog’, ‘frog’, ‘horse’, ‘ship’, ‘truck’. The images in CIFAR-10 are of
 size 3x32x32, i.e. 3-channel color images of 32x32 pixels in size.
 
-.. figure:: /_static/img/cifar10.png
-   :alt: cifar10
+![](https://pytorch.org/tutorials/_images/cifar10.png)
```

Comment:
Why this is needed? (I.e. keeping it as is allows one to re-host the repository anywhere they want, isn't it?)